<ui version="4.0" >
 <class>MainFormBase</class>
 <widget class="QWidget" name="MainFormBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PeerCast</string>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>6</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>6</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_4" >
            <property name="font" >
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text" >
             <string> PeerCast-VP </string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonEnabled" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>3</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Enabled</string>
            </property>
            <property name="checkable" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>2</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="2" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Max relays</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="lineEditPort" >
            <property name="alignment" >
             <set>Qt::AlignRight</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Password</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="lineEditPassword" >
            <property name="echoMode" >
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Port</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="3" >
           <widget class="QLineEdit" name="lineEditMaxRelays" >
            <property name="alignment" >
             <set>Qt::AlignRight</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth" >
      <number>4</number>
     </property>
     <widget class="QWidget" name="layoutWidget_2" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>4</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButtonPlay" >
           <property name="text" >
            <string>Play</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButtonBump" >
             <property name="text" >
              <string>Bump</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonDisconnect" >
             <property name="text" >
              <string>Disconnect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonKeep" >
             <property name="text" >
              <string>Keep</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Q3ListBox" native="1" name="listBoxChannel" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_3" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>4</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButtonDisconnectConn" >
           <property name="text" >
            <string>Disconnect</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Q3ListBox" native="1" name="listBoxConnection" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_4" >
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>4</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButtonClear" >
           <property name="text" >
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonStop" >
           <property name="text" >
            <string>Stop</string>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButtonDebug" >
             <property name="sizePolicy" >
              <sizepolicy>
               <hsizetype>1</hsizetype>
               <vsizetype>0</vsizetype>
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="sizeIncrement" >
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="baseSize" >
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text" >
              <string>Debug</string>
             </property>
             <property name="checkable" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonError" >
             <property name="text" >
              <string>Error</string>
             </property>
             <property name="checkable" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonNetwork" >
             <property name="text" >
              <string>Network</string>
             </property>
             <property name="checkable" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonChannel" >
             <property name="text" >
              <string>Channel</string>
             </property>
             <property name="checkable" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Q3ListBox" native="1" name="listBoxLog" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Q3ListBox</class>
   <extends>Q3Frame</extends>
   <header>q3listbox.h</header>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QWidget</extends>
   <header>q3frame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonEnabled</tabstop>
  <tabstop>lineEditPort</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>lineEditMaxRelays</tabstop>
  <tabstop>pushButtonPlay</tabstop>
  <tabstop>pushButtonBump</tabstop>
  <tabstop>pushButtonDisconnect</tabstop>
  <tabstop>pushButtonKeep</tabstop>
  <tabstop>listBoxChannel</tabstop>
  <tabstop>pushButtonDisconnectConn</tabstop>
  <tabstop>listBoxConnection</tabstop>
  <tabstop>pushButtonClear</tabstop>
  <tabstop>pushButtonStop</tabstop>
  <tabstop>pushButtonDebug</tabstop>
  <tabstop>pushButtonError</tabstop>
  <tabstop>pushButtonNetwork</tabstop>
  <tabstop>pushButtonChannel</tabstop>
  <tabstop>listBoxLog</tabstop>
  <tabstop>pushButtonPlay</tabstop>
  <tabstop>pushButtonBump</tabstop>
  <tabstop>pushButtonDisconnect</tabstop>
  <tabstop>pushButtonKeep</tabstop>
  <tabstop>listBoxChannel</tabstop>
  <tabstop>pushButtonDisconnectConn</tabstop>
  <tabstop>listBoxConnection</tabstop>
  <tabstop>pushButtonClear</tabstop>
  <tabstop>pushButtonStop</tabstop>
  <tabstop>pushButtonChannel</tabstop>
  <tabstop>pushButtonDebug</tabstop>
  <tabstop>pushButtonError</tabstop>
  <tabstop>pushButtonNetwork</tabstop>
  <tabstop>listBoxLog</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
